### ISSUE TITLE
[01_MSPM_Pretraining.ipynb] nan valid_loss when doing learner.lr_find

### ISSUE THREAD

**Issue by riven314** [XinhaoLi74/MolPMoFiT#2]
Thanks for such a great work!

I am trying to replicate your work and by the time I was running ```01_MSPM_Pretraining.ipynb```, I found that valid_loss got ```#na#``` when executing ```learner.lr_find()```, as shown below: 

![nan_valid_loss](https://user-images.githubusercontent.com/21143399/75090834-fd8a0d00-55a1-11ea-9514-db9e0bb547f1.JPG)

I am figuring out the cause. Did you encounter that in your experiment?

**Comment by riven314** [comment]
Besides that, I also noticed that you avoid dropout when training language model (i.e. ```learner = language_model_learner(data_lm, AWD_LSTM, drop_mult=1, pretrained=False)``` ). Is it because dropout leads to worse performance?

**Comment by XinhaoLi74** [comment]
> Besides that, I also noticed that you avoid dropout when training language model (i.e. `learner = language_model_learner(data_lm, AWD_LSTM, drop_mult=1, pretrained=False)` ). Is it because dropout leads to worse performance?

In the [configuration](https://github.com/fastai/fastai/blob/master/fastai/text/models/awd_lstm.py) of the AWD_LSTM model, the dropout is already applied in the model. Specifically, `hidden_p:float=0.2, input_p:float=0.6, embed_p:float=0.1, weight_p:float=0.5`. `drop_mult` is the value to multiply the dropout values in the configuration. `drop_mult=1` means use the same dropouts as those in the configuration of the model. 

The performance of the pre-trained MSPM is not important. The value of the pre-trained model is determined by the performances of models on the downstream tasks. In my experiments, I tested the `drop_mult` values of 0.2 and 1. It didn't affect too much.

> I am figuring out the cause. Did you encounter that in your experiment?
I am not sure about this. Did you encounter the same problem when you train the model?

**Comment by riven314** [comment]
@XinhaoLi74  
Thanks for the prompt address! 
For the nan issue, so far I didn't encounter that in training stage. It only occurred in ```learner.lr_find()```. Then I shall regard it as a minor issue.
